package.json

```diff
@@ -1,23 +1,58 @@
 {
-    "name": "html-flip-book",
-    "version": "1.0.0",
-    "description": "Flipbook component for HTML",
-    "scripts": {},
-    "author": "DoradSoft",
-    "license": "MIT",
-    "devDependencies": {
-        "css-loader": "^7.1.1",
-        "eslint": "^9.3.0",
-        "mini-css-extract-plugin": "^2.9.0",
-        "sass": "^1.77.2",
-        "sass-loader": "^14.2.1",
-        "style-loader": "^4.0.0",
-        "ts-loader": "^9.5.1",
-        "ts-node": "^10.9.2",
-        "typescript": "^5.4.5",
-        "typescript-eslint": "^8.0.0-alpha.14",
-        "vite": "^6.0.0-alpha-15",
-        "webpack": "^5.91.0",
-        "webpack-cli": "^5.1.4"
-    }
+  "name": "html-flip-book",
+  "version": "1.0.0",
+  "description": "Flipbook component for HTML",
+  "homepage": "https://github.com/doradsoft/html-flip-book",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/doradsoft/html-flip-book.git"
+  },
+  "bugs": {
+    "url": "https://github.com/doradsoft/html-flip-book/issues"
+  },
+  "engines": {
+    "node": ">=22.2.0"
+  },
+  "scripts": {
+    "bundle": "npm run format:write && npm run package",
+    "ci-test": "npx jest --all --ci --coverage --coverageDirectory ./coverage",
+    "coverage": "npx make-coverage-badge --output-path ./badges/coverage.svg",
+    "format:write": "npx prettier --write .",
```

Comment:
prettier is not properly configured IMO so the restyling this make is not desirable for now, if you want you can invest in making this better